src/components/home/vacations-card.tsx

```diff
@@ -143,11 +149,59 @@ const VacationsCard = () => {
     fetchVacationsRequests();
   }, []);
 
+  /**
+   * Initialize logged in person's data.
+   */
+  const getPersonVacations = async () => {
```

Comment:
This function name could be reduced to "getPersons", as this is what the function seems to do.

Reply:
Changed the functions name.